dsl/docker-multiarch/images/solr.groovy

```diff
@@ -0,0 +1,38 @@
+import vars.multiarch
+
+for (arch in multiarch.allArches()) {
+        meta = multiarch.meta(getClass(), arch)
+
+        freeStyleJob(meta.name) {
+                description(meta.description)
+                logRotator { daysToKeep(30) }
+                label(meta.label)
+                scm {
+                        git {
+                                remote { url('https://github.com/docker-solr/docker-solr') }
+                                branches('*/master')
+                                clean()
+                        }
+                }
+                triggers {
+                        upstream("docker-${arch}-openjdk", 'UNSTABLE')
+                        scm('H H/6 * * *')
+                }
+                wrappers { colorizeOutput() }
+                steps {
+                        shell(multiarch.templateArgs(meta) + '''
+sed -i "s!^FROM    !FROM    $prefix/!" */Dockerfile
```

Comment:
Think it's worth using something like `sed -i "s!^FROM +!FROM $prefix/!" */Dockerfile` instead so we capture the spaces no matter how many there are? (in case it changes)